guides/common/modules/proc_assigning-a-host-to-a-specific-organization.adoc

```diff
@@ -2,11 +2,12 @@
 = Assigning a host to a specific organization
 
 Use this procedure to assign a host to a specific organization.
+For general information about organizations and how to configure them,
 ifdef::satellite[]
-For general information about organizations and how to configure them, see {AdministeringDocURL}Managing_Organizations_admin[Managing Organizations] in _{AdministeringDocTitle}_.
+see {AdministeringDocURL}Managing_Organizations_admin[Managing organizations] in _{AdministeringDocTitle}_.
 endif::[]
 ifndef::satellite[]
-For general information about organizations and how to configure them, see {ManagingOrganizationsLocationsDocURL}Managing_Organizations_managing-organizations-locations[Managing Organizations] in _{ManagingOrganizationsLocationsDocTitle}_.
+see {ManagingOrganizationsLocationsDocURL}Managing_Organizations_managing-organizations-locations[Managing organizations] in _{ManagingOrganizationsLocationsDocTitle}_.
```

Comment:
Similar point as the one above. You could also link to the whole guide.

Reply:
Applied.